grafana/Dockerfile.dev

```diff
@@ -0,0 +1,26 @@
+FROM gliderlabs/alpine:3.1
```

Comment:
nvm. I see a testing use case below. Should we move to this under a '/test' directory?